models/employeesModel.js

```diff
@@ -15,5 +15,10 @@ const employeesSchema = mongoose.Schema({
   suspended: { type: Boolean, default: false },
   phone: { type: Number, required: true, unique: true },
 });
-
+employeesSchema.index({
```

Comment:
Is this neccessary??

Reply:
remove it please!!!

Reply:
found it unnecessary and removed it. thank